.devcontainer/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM mcr.microsoft.com/devcontainers/go:1-1.21-bookworm
```

Comment:
Any reason for not using this instead of https://hub.docker.com/_/golang?

Reply:
For devcontainers one needs to use the devcontainers root image instead of the official one